tide/master/hardware/MultiScreenController.cpp

```diff
@@ -42,6 +43,40 @@
 #include "PlanarController.h"
 #include "configuration/Configuration.h"
 
+namespace
+{
+ScreenState _getResult(const std::vector<ScreenState>& results)
+{
+    const auto state = results[0];
```

Comment:
what if results empty?

Reply:
Right, this function should be safe even though it normally doesn't happen in this context, updated.